main.tf

```diff
@@ -108,6 +108,7 @@ resource "google_cloudfunctions2_function" "webhook" {
       DOCAI_LOCATION    = google_document_ai_processor.ocr.location
       DATABASE          = google_firestore_database.main.name
       INDEX_ID          = google_vertex_ai_index.docs.id
+      LOG_EXECUTION_ID  = true
```

Comment:
Reason for this `LOG_EXECUTION_ID = true` line:
The verify step in our integration tests were failing:
```
    module.simple.google_cloudfunctions2_function.webhook will be updated in-place
    "google_cloudfunctions2_function" "webhook" {
        service_config {
            environment_variables            = {
                "LOG_EXECUTION_ID"  = "true" -> null
…
Plan: 0 to add, 1 to change, 0 to destroy.
```

It sounds like Google Cloud (under the hood) sets this LOG_EXECUTION_ID environment variable to true — after we terraform apply. So subsequent "terraform apply"s think the Cloud Function needs to be updated since, in our Terraform definition, the value is null (not set).